.ci/windows-steps.yaml

```diff
@@ -48,7 +45,7 @@ steps:
     msbuildVersion: $(MSBuildVersion)
     configuration: 'Release'
     msbuildArchitecture: 'x64'
-    msbuildArguments: /m /p:BuildInParallel=true
+#    msbuildArguments: /m /p:BuildInParallel=true
```

Comment:
Building Armadillo in parallel would probably be fine, so maybe we can uncomment this?  But I don't think it matters much.

Reply:
Yes, of course, it was only for precaution:+1: